docs/sections/pipeline_samples/tutorials/GenerateSentencePair.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    llm = OpenAILLM(model="gpt-4o", api_key=os.getenv("OPENAI_API_KEY"))
</pre><p>Could we use <code>InferenceEndpointsLLM</code> instead? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/893/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<img width="1460" alt="image" src="https://github.com/user-attachments/assets/7f4a0406-0901-4b53-a43a-66e988e90683">